app/Models/Hit.php

```diff
@@ -22,9 +22,19 @@ class Hit extends Model
     public static function forTimePeriod(string $period = 'week'): array
     {
         $now = CarbonImmutable::now();
+        $currentPeriodStart = $now->sub($period, 1)->addSecond();
+        $previousPeriodEnd = $currentPeriodStart->subSecond();
+        $previousPeriodStart = $previousPeriodEnd->sub($period, 1)->addSecond();
 
```

Comment:
Could these instead be this? A little tweak that makes it a bit easier for me to follow it.

```suggestion
$currentPeriodEnd = CarbonImmutable::now();
$previousPeriodEnd = $currentPeriodEnd->sub($period, 1);
$currentPeriodStart = $previousPeriodEnd->addSecond();
$previousPeriodStart = $previousPeriodEnd->sub($period, 1)->addSecond();
```